### DISCUSSION TITLE
Open select with .click() on element

### DISCUSSION THREAD

**Discussion by ghost** [JedWatson/react-select#5285] [Q&A]
Hi guys,
Is it possible to open react-select with .click() on the element?
document.querySelector('.Select-control"]').click(); or something like that?

**Comment by cojoclaudiu** [comment]
Add a ref and after call ref.current.click()

**Comment by manjushsh** [comment]
@Zvih you can use ref to focus and open on focus using flag: `openMenuOnFocus={true}`
Something like this:
`<Select ref={selectRef} options={options} openMenuOnFocus={true} />`

[Example to open select on button click on Code Sandbox](https://codesandbox.io/s/react-select-multiple-old-forked-6picqk?file=/src/reffed-select.tsx)

**Comment by twigs67** [comment]
This tells you how to do it via the react component, but how would you do it if you wanted to .click() from the console? @manjushsh @cojoclaudiu

**Reply by twigs67** [reply]
Any update? @manjushsh

**Reply by manjushsh** [reply]
No man. I cant figure out how to do it through console.
